package.json

```diff
@@ -67,6 +69,7 @@
     "jest-md-dashboard": "0.8.0",
     "knip": "5.27.0",
     "lint-staged": "^15.2.2",
+    "nock": "13.5.6",
```

Comment:
nock for http mocking here? We are already using msw in `text-conversation-rewards` plugin and `pay.ubq.fi`.

Reply:
Usually we mock all API responses but I built the tests specifically so that we mocked as little as possible which is why we have a local `Anvil` instance for write ops and use the real public nodes in other tests. 

I think typically you shouldn't use the real responses but I felt it was a better way to test this package so I wasn't potentially fooling myself. 

Nock was quick and easy with little overhead and because it's only used for one test suite I didn't think it would be too much of an issue.